test/functional/services/query_bar.ts

```diff
@@ -54,6 +54,11 @@ export function QueryBarProvider({ getService, getPageObjects }: FtrProviderCont
       });
     }
 
+    public async clearQuery(): Promise<void> {
+      await this.setQuery('');
+      await PageObjects.common.pressTabKey();
```

Comment:
moving focus by tab closes the query bar suggestions, also the time picker is then displayed again, which simplifies the code of the failing Discover test